src/Item.h

```diff
@@ -141,8 +141,9 @@ class cItem
 	int GetEnchantability();  // tolua_export
 
 	/** Randomly enchants the item using the specified number of XP levels.
-	Returns true if the item was enchanted, false if not (not enchantable / too many enchantments already). */
-	bool EnchantByXPLevels(int a_NumXPLevels);  // tolua_export
+	Returns true if the item was enchanted, false if not (not enchantable / too many enchantments already).
+	Passing an MTRand PRNG pointer is optional. */
+	bool EnchantByXPLevels(int a_NumXPLevels, MTRand * Random = nullptr);  // tolua_export
```

Comment:
MTRand ain't exported to Lua, could you make this MTRand & and export manually in ManualBindings.cpp?

Reply:
Is there a way to only export the function with the default parameter and prevent them from passing the random generator? I don't think they need access to MTRand (and I've no clue how it's done).

Reply:
Let me help with the export